Add FAQ about Omitted Properties (#4617)
segmentio/segment-docs: src/protocols/faq.md

Unfortunately, Sources cannot be connected to more than one Tracking Plan. If you were able to connect more than one Tracking Plan to a Source, it could create conflict if events overlapped. 

### Why are my unplanned properties still getting sent to my destinations even though I've set the dropdown to "Omit Properties"?

> Unplanned property omission is only supported in cloud-mode destinations. Unplanned properties will not be omitted when sending to device-mode destinations.

Reviewer: ```suggestion
Unplanned property omission is only supported for cloud-mode destinations. Unplanned properties will not be omitted when they're sent to device-mode destinations.
```
